src/V3AstNodes.cpp

```diff
@@ -706,6 +706,43 @@ class AstNodeDType::CTypeRecursed final {
     }
 };
 
+string AstNodeDType::cTypeFromWidth() const {
+    if (width() <= 8)
+        return "CData";
+    else if (width() <= 16)
+        return "SData";
+    else if (width() <= VL_IDATASIZE)
+        return "IData";
+    else if (isQuad())
+        return "QData";
+    else
+        return "VlWide<" + cvtToStr(widthWords()) + ">";
+}
+
+string AstNodeDType::cpackedType(const string& name) const {
+    const CTypeRecursed info = cpackedTypeRecurse();
+    return info.render(name, false);
+}
+
+AstNodeDType::CTypeRecursed AstNodeDType::cpackedTypeRecurse() const {
+    CTypeRecursed info;
```

Comment:
Would this be better as an option or otherwise in the existing function below to avoid duplicate code?

Reply:
Have added a agrument `packed` to cTypeRecurse() and merged these two functions